app/experimenter/nimbus-ui/src/hooks/useFakeMutation.tsx

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { useCallback, useState } from "react";
+
+export const useFakeMutation = (delay = 1000) => {
```

Comment:
Figured adding this as a reusable hook would be handy for more of these sorts of front-end commits in advance of back-end support.

Reply:
This is neat. Should it take a callback, so something like a Storybook Action could be passed in?

Reply:
Could be handy. Not sure how to plumb an action in there, though. The fake mutations are instantiated in the page component. Maybe some temporary page-level props, but those should get pulled out at polish time

Reply:
Ohhh yup! Makes sense. Carry on.